cypress/integration/latest-epochs-on-homepage.ts

```diff
@@ -0,0 +1,12 @@
+describe('Latest Epochs on Homepage', () => {
```

Comment:
```suggestion
describe('Homepage data feeds', () => {
```
Let's keep consistent with the original feature spec: https://github.com/input-output-hk/cardano-explorer-app/blob/spec/ddw-884-mvp-feature-specifications/features/home-page-data-feeds.feature